state_machine/drivers/pycubedmini/lib/pycubed.py

```diff
@@ -424,6 +424,18 @@ def sun_vector(self):
              self.sun_yp.lux - self.sun_yn.lux,
              self.sun_zp.lux - self.sun_zn.lux])
 
+    @property
```

Comment:
I'm not seeing how these functions (`lux_p`, and `lux_n`) add anything useful to what we already have in pycubed.py, let's leave them out.